includes/Settings.php

```diff
@@ -424,6 +424,7 @@ public function generate_import_products_html( $id, $field ) {
 				<a id="wc_square_import_products" href='#' class='button js-import-square-products <?php echo ( ! $this->get_location_id() ? 'disabled' : '' ); ?>'>
 					<?php echo esc_html__( 'Import all products from Square', 'woocommerce-square' ); ?>
 				</a>
+				<p class="description" style="margin-top: 8px;"><?php esc_html_e( 'Use this to bring new products from Square into WooCommerce. This is different from "Sync Now" which only updates products that are already synced.', 'woocommerce-square' ); ?></p>
```

Comment:
"This is different from "Sync Now" which only updates products that are already synced." This is correct only when SOR is Square.